src/kakarot/execution_context.cairo

```diff
@@ -46,6 +51,8 @@ namespace ExecutionContext {
         let memory: model.Memory* = Memory.init();
 
         local ctx: model.ExecutionContext* = new model.ExecutionContext(
+            sn_address=sn_contract_address,
```

Comment:
could we rename this starknet_contract_address or starknet_address to keep consistency in the repo?